app/src/main/java/org/oppia/android/app/databinding/MarginBindingAdapters.java

```diff
@@ -1,22 +1,35 @@
 package org.oppia.android.app.databinding;
 
+import android.annotation.SuppressLint;
 import android.view.View;
 import android.view.ViewGroup.MarginLayoutParams;
 import androidx.annotation.NonNull;
+import androidx.core.view.ViewCompat;
 import androidx.databinding.BindingAdapter;
 
 /** Holds all custom binding adapters that set margin values. */
 public final class MarginBindingAdapters {
 
+  @SuppressLint("WrongConstant")
+  private static boolean isRtlLayout(View view) {
+    return view.getLayoutDirection() == ViewCompat.LAYOUT_DIRECTION_RTL;
```

Comment:
@veena14cs Has this been confirmed by @BenHenning ? The use of `LAYOUT_DIRECTION_RTL ` and also `SuppressLint` ?

Reply:
@BenHenning can you please look into this.

Reply:
We should be using https://developer.android.com/reference/androidx/core/view/ViewCompat#getLayoutDirection(android.view.View) since that will work on older Android versions.

I suspect that will also fix the lint issue and that we can then remove the suppression.

This approach seems reasonable given that it's the only clear way to check whether the current layout is RTL.

Reply:
Thanks. Done this change.

Reply:
Thanks.